geometry_msgs/Pose2D tail_pose
float64[] joint_angles_rad
